special-interest-groups/sig-tiup/member-list.md

```diff
@@ -0,0 +1,27 @@
+# SIG TiUP Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: N/A
+
+## Tech Leads
+
+* [lonng](https://github.com/lonng), [PingCAP](https://pingcap.com/en/)
+* [lucklove](https://github.com/lucklove), [PingCAP](https://pingcap.com/en/)
+
+## Committers
+
+None
+
+## Reviewers
+
+None
+
+## Active Contributors
+
+Members and their contributions:
+
+|                   user                    | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
```

Comment:
I think no needs to show number of merged PRs and earliest_PR_merged_time and latest_PR_merged_time. Because they are dynamic. It will waste much time to maintain.